### ISSUE TITLE
"Autocomplete: value missing" message

### ISSUE THREAD

**Issue by eduardabp** [PhilippRecke/Autocomplete-Check#6]
The message "Autocomplete: value missing" can be confusing for users. When I think in value, I think on `key:value` pair, so it would make me think that there is an `autocomplete`, but with no value assigned to it. On this case, it is being used to say that there is no `autocomplete` attribute at all.

**Comment by PhilippRecke** [comment]
Yeah, good point. How would you like changing the text to something like "No autocomplete". Short but sufficient? :)
